ListingManager/ListingManager.cs

```diff
@@ -59,25 +60,47 @@ private static bool TryGetListing(string listingPath, out ListingInformation lis
 
                 var curListingData = listingData[i];
 
-                if (listingNumber == curListingData.ListingNumber)
+                if (chapteronly == false && byfolder == false)
+                {
+                    if (listingNumber == curListingData.ListingNumber) //default
```

Comment:
Instead of nesting these lets just collapse this and add this condition to the outer if block. I try to avoid nesting whenever possible